types/preview/ember/index.d.ts

```diff
@@ -0,0 +1,381 @@
+declare module 'ember' {
+  // Capitalization is intentional: this makes it much easier to re-export RSVP on
+  // the Ember namespace.
+  import Rsvp from 'rsvp';
+
+  import type { Registry as ControllerRegistry } from '@ember/controller';
+  import type * as EmberStringNs from '@ember/string';
+  import type * as EmberTemplateNs from '@ember/template';
+  import * as EmberTemplateHandlebarsNs from '@ember/template/-private/handlebars';
+  import type * as EmberServiceNs from '@ember/service';
+  import type * as EmberPolyfillsNs from '@ember/polyfills';
+  import type * as EmberUtilsNs from '@ember/utils';
+  import type * as EmberRunloopNs from '@ember/runloop';
+  import * as EmberObjectNs from '@ember/object';
+  import type * as EmberObjectObserversNs from '@ember/object/observers';
+  import * as EmberObjectProxyNs from '@ember/object/proxy';
+  import type * as EmberObjectPromiseProxyNs from '@ember/object/promise-proxy-mixin';
```

Comment:
It's roughly: "whatever VS Code + ESLint happened to autofix when I was otherwise fixing up imports." 🙃 We can and should make sure we end up with a consistent style if/as we iterate on the ambient types between now and publishing types from source.